doc/source/tune/_tutorials/tune-60-seconds.rst

```diff
@@ -72,70 +69,78 @@ Use ``tune.run`` execute hyperparameter tuning using the core Ray APIs. This fun
     # Pass in a Trainable class or function to tune.run.
     tune.run(trainable)
 
-    # Run 10 trials (each trial is one instance of a Trainable). Tune runs in
-    # parallel and automatically determines concurrency.
-    tune.run(trainable, num_samples=10)
-
-    # Run 1 trial, stop when trial has reached 10 iterations OR a mean accuracy of 0.98.
-    tune.run(my_trainable, stop={"training_iteration": 10, "mean_accuracy": 0.98})
-
-    # Run 1 trial, search over hyperparameters, stop after 10 iterations.
-    hyperparameters = {"lr": tune.uniform(0, 1), "momentum": tune.uniform(0, 1)}
-    tune.run(my_trainable, config=hyperparameters, stop={"training_iteration": 10})
-
 This function will report status on the command line until all Trials stop:
 
 .. code-block:: bash
 
     == Status ==
     Memory usage on this node: 11.4/16.0 GiB
     Using FIFO scheduling algorithm.
-    Resources requested: 4/12 CPUs, 0/0 GPUs, 0.0/3.17 GiB heap, 0.0/1.07 GiB objects
+    Resources requested: 1/12 CPUs, 0/0 GPUs, 0.0/3.17 GiB heap, 0.0/1.07 GiB objects
     Result logdir: /Users/foo/ray_results/myexp
-    Number of trials: 4 (4 RUNNING)
+    Number of trials: 1 (1 RUNNING)
     +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
     | Trial name           | status   | loc                 |    param1 | param2 |    acc | total time (s) |  iter |
     |----------------------+----------+---------------------+-----------+--------+--------+----------------+-------|
     | MyTrainable_a826033a | RUNNING  | 10.234.98.164:31115 | 0.303706  | 0.0761 | 0.1289 |        7.54952 |    15 |
-    | MyTrainable_a8263fc6 | RUNNING  | 10.234.98.164:31117 | 0.929276  | 0.158  | 0.4865 |        7.0501  |    14 |
-    | MyTrainable_a8267914 | RUNNING  | 10.234.98.164:31111 | 0.068426  | 0.0319 | 0.9585 |        7.0477  |    14 |
-    | MyTrainable_a826b7bc | RUNNING  | 10.234.98.164:31112 | 0.729127  | 0.0748 | 0.1797 |        7.05715 |    14 |
     +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
 
+
+You can also easily run 10 trials (each trial is one instance of a Trainable). Tune automatically determines parallelism.
```

Comment:
How does it determine parallelism? Where can I learn more about this? Include links...